docs/ota-client-guide/modules/ROOT/pages/upload-large-binary.adoc

```diff
@@ -0,0 +1,79 @@
+= Upload a binary file
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+If you want to upload large binary files (up to 3 GB) or your software signing keys are xref:ota-client::rotating-signing-keys.adoc[taken offline], use the `garage-sign` tool on the command line.

```

Comment:
```suggestion
If you want to upload large binary files (up to 3 GB) or your software signing keys are xref:ota-client::rotating-signing-keys.adoc[offline], use the `garage-sign` tool on the command line.
```

Alternatively, "have been taken offline" works too, but it's more verbose.

Reply:
Changed